src/test/java/core/basesyntax/services/operations/FruitOperationStrategyAndFruitTransactionProcessorTest.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax.services.operations;
+
+import static core.basesyntax.db.Storage.fruitStorage;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dto.FruitTransactionDto;
+import core.basesyntax.exceptions.FruitStorageException;
+import core.basesyntax.services.FruitTransactionProcessor;
+import core.basesyntax.services.impl.FruitTransactionProcessorImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FruitOperationStrategyAndFruitTransactionProcessorTest {
+
+    private static FruitOperationStrategy strategy;
+    private static List<OperationHandler> handlers;
```

Comment:
let's look closer just at the strategy class, and what methods it has, it's just `getHandler()` and so that's what we should test here, handler tests should also be separate, and we should test not just `PurchaseHandler`, but all of them should be tested, at least with positive cases